compiler/IREmitter/IREmitterHelpers.cc

```diff
@@ -206,7 +205,7 @@ void IREmitterHelpers::emitReturnAcrossBlock(CompilerState &cs, cfg::CFG &cfg, l
 void IREmitterHelpers::emitReturn(CompilerState &cs, llvm::IRBuilderBase &builder, const IREmitterContext &irctx,
                                   int rubyRegionId, llvm::Value *retVal) {
     if (functionTypeNeedsPostprocessing(irctx.rubyBlockType[rubyRegionId])) {
-        auto returnValue = irctx.cfg.enterLocal({Names::returnValue(cs), 1});
+        auto returnValue = irctx.cfg.enterLocal({Names::returnValue(), 1});
```

Comment:
The necessity for this name is super gross, but maybe that can be addressed in a future change (not incumbent upon you to fix this in any way)!